src/extendedstorage.cpp

```diff
@@ -696,7 +680,7 @@ Incidence::Ptr ExtendedStorage::checkAlarm(const QString &uid, const QString &re
 
 Notebook::Ptr ExtendedStorage::createDefaultNotebook(QString name, QString color)
 {
-    QString uid;
+    QByteArray uid(QUuid::createUuid().toByteArray());
```

Comment:
Looks like existing thing with both notebooks and incidences with createUuid() but not following what's this byte array conversion. Both notebook and incidencebase use string as type for the uid?

Reply:
Oops, yes, one can use `.toString()` instead of `.toByteArray()`.